### ISSUE TITLE
How the website should look?

### ISSUE THREAD

**Issue by girlscript** [girlscript/contribute-girlscript#3]
This is the overview of the entire website and what it entails:

 
![ss1](https://cloud.githubusercontent.com/assets/27850350/26229942/e3aab142-3c62-11e7-8f78-f65310fc8300.PNG)
What all pages should the website contain?
 What should be the basic theme of the website? - design, background, color combination
 Basic mock-up of the home page.

**Comment by siddhartharao17** [comment]
The picture you have published above looks pretty good as the homepage. Why don't we do it exactly like this?

**Comment by girlscript** [comment]
Yes, we are working on it. This screenshot is website built by @anubhamane. Although, we are looking for more suggestions to make it better
